loculus_values/environment_specific_values/demo.yaml

```diff
@@ -14,7 +14,6 @@ secrets:
     encryptedData:
       backendUserPassword: AgCQqDEX355jCx7+wqkeIF2ssxZQC+JOB9GnJANMzXpKQuTBB0IV834lvAiJVWEhfrrkzF9V6reftt+wqV/G+2xZxCVUM55l+ySR+/EtJASOdt1aj3jexY+0uTKV5PlqXlxNhnwpPdUsqyPm2NViHkszdwmKr12eD7R5lwOl2pqrhKRAXpFRsYRoxiz9yL+Bz2m+y7+OMyI3R5Y1A74OBm8tdUQgCpoWK0SnEYHiO/yOlUVHLryTxHwEgeJjOQSMrd3NAy6qiSYWq//+4hL9Yq6TcQbk6Z62SclhOtpTvaMr08AqnimwvakHtSPMNlNmg3B9dkXJw6Ol8uX8QuU85EMWKTwKptyWQM4SbYqRp5xZzoaJYKpHCtyQU5kFcOw8gYAMNjqGqo8uAAIyk4tbYh5aFAGOx4SiBHLvCyTQlFgUGj6lKFpWkxIV3JjTAX7lL1FmVZrqziXprZgWpGnlMg7ZhTVzPd65OuJ6fgbnPjsZceB/B/O0nAPA8ySPlaFgH8ZclWCaoF+ByRq3WNVflyVf1jegzsVas8qkpdDFMGTqY4lyNzM6hqNx6ssM30evL3zgufOTIsY07Gc3LKNYQ3SQSjGwRMGQ7719S7KVIxeFdKQzFJlL52y1/D8aupeiYwEOlRvlRRjiYzhObtTCHhFeliVT3BWnPEX7QteLdxe9FduXzl9OWPiVsv2Wo4qEsU6Phs72d4RiihVrOUl6nhBRwVVQ3uVYKKbSOc2hxAvcCg==
       externalMetadataUpdaterPassword: AgBUhqaP8IuUff3xes1M85JWb8DTjsUf3kvId27vOKtprmOupeuP/tfuRI4CY7DEuc0Ksg+NX230FTa4HauQzzyh48BCxgaTytXNR/VxCYEKVqM+PJtJP2htbpEW7vrJm/woGx67O2pAJBgELd2LfUUHQoEh/nApT4JCo3bjPfEY4/FeVcxUnZyCX7o0NvxvaPiOf348ZYpuxCBTyfo4cfBGLqvq/FtvLWNqz7Yf2nO+Dfz5rbIl+Z7IIVhBONOZDQUOt4Slk4CzA/QEQbfSa4T2RaVSfkVduQM9cv/H9GCUx6kCWXh2EjYKOV5kjlOFDGEwv+gNKhIZPPLjxMxeceIIqbNAmi8T0oPxUUTGNmI+AYSIaD3FQo9XWgD61ti4bI/nfrprKPJLv4TzkrOK6MWO0CKG1p/uugYQDgWA+KveIkGRGD8lPudDTnIQKUZRby8xALjEr9P+KraqEXEl+gCuN9GkQtdvT2wovmjCgBFCdJ2d8cXAoBxaoFouBTEO7Nil3nJ5FM8wPYvlY6bV6UHXEbQDMmxMjoRS5RNYaoizNWB2n6FmxllDvaj7OIsyjvkcEH1MRMLXwSlfpd7PHMWyjEKQISZ+3alV0EO/nwzv4r9Z/DkYToH01IiNGitMcpgbaI21nkJdOJyRlAkLVtN2qoi4DBL9siElbCL8Tt/W/yCzctz5T/PGUraHKE5EddWXWVpTTtdzjTTMVRAT6H+r9TOsEJ+rxR/VrDR1wVQcJw==
-      insdcIngestUserPassword: AgAHkYcaqcK64wwpzA1obwlXTMZzVoZh1eosbo8+dDIIaGP7msTuucKKILfj7lcj3rBIvu+r6k9PZtkTrVhjCIApM9DuaZTta86keNRu4gbRWkTZtoNEY/q1mR5HNsWTJ6RGd9+4yedXP8kEpTiJw2dSV7QUTqTqGIsj0Gm01h6TFsg1dYxpXtc1EEVMgtTPAe2lNwN5+H2Y89bL3V8OjElBmMy2rVJRIDjF226CJ/WiucRXOPPRwU2G/ACvDP8KjGftJrkEd357MrsBNEG7WyMq+0G4iCtN5kLqTsING5IhHYqEsOUMKy89z4sK23VDmkCeeukhfknuWnrcUd+VAvJM18mvkzl/t7n5aDXV7AGfXYJ0uuqD6gfMEgjrdWCm2vN3V25spMMj1wqwZYtaAvpP4+DKxusum6WHiRz65DBznJehDhqdVfKEaqJXrET4DUA1rjmX4Yv+9CBulbVf286BaVu69Y/OPaNv8tLVCmIx6+Vn3LwZotP+rdxPYCytnVB2ewVKJOKTvt6vZei4RV3u5Ry6IgmK6x03BRKpbWE4LD36ujzgK92DtzOpVEGJ4F9ybwDS4CDrXv7r4fJLXo4KiGRtbnRVHzij8lezFfjPx7E45jrzQc3r5YYgNGe7Sl8NGLnGGJ0fU4HAnF6VZFIdFqca8xlMT3miyIIlnMBsh7tsU3IBM15vDymaMxSE8u46dKH3j47Zc0bx6HfBaVeRbFOhPokGPQFf30WUPL7Eig==
```

Comment:
how come?

Reply:
I think ingest still needs a password? it asks for get-original-metadata and this still requires a password

Reply:
Yeah indeed, sorry for screwing this up, thanks for spotting!

Reply:
Should be fixed now I hope 🙃